src/Microsoft.AspNetCore.WebHooks.Receivers/ApplicationModels/WebHookActionModelPropertyProvider.cs

```diff
@@ -50,60 +55,62 @@ public class WebHookMetadataProvider : IApplicationModelProvider
         /// <param name="verifyCodeMetadata">
         /// The collection of <see cref="IWebHookVerifyCodeMetadata"/> services.
         /// </param>
-        public WebHookMetadataProvider(
+        public WebHookActionModelPropertyProvider(
```

Comment:
I think we should spend some time talking through this. This is more indirection then I expected to see here and I'd like to understand it.

Reply:
As above, decided to leave this as-is.